packages/nx-electron/package.json

```diff
@@ -47,7 +47,7 @@
   ],
   "peerDependencies": {
     "electron": "*",
-    "@nx/workspace": "^16.0.0"
+    "@nx/workspace": ">=16"
```

Comment:
Please update to "^16"
I will publish a new version for v17

Reply:
@bennymeg Is there anything I can assist with to publish the new version? Due to this change, I am unable to use the package with Nx 17

Reply:
there is @kockicica/nx-electron npm package as quick and dirty fix until proper nx17 version is released.

Reply:
@brnbs could you try `nx-electron@17.0.0-alpha.0`?

Reply:
@bennymeg 17.0.0-alpha.0 seems to work for me with Nx 17.0.2
@kockicica thanks for sharing!